scripts/api-docs.mustache

```diff
@@ -0,0 +1,164 @@
+# Firefox Accounts authentication server API
```

Comment:
Let's add a pre-push hook to run the api doc script. It can work the same way as mailer templates (e.g if there is a change then push will be blocked and the new docs need to be committed)